operations/package_repository_management/Manage_Repositories_with_Nexus.md

```diff
@@ -370,6 +394,12 @@ The `proxy`, `httpClient`, and `negativeCache` options impact the proxy behavior
 Installers typically define Nexus repositories in `nexus-repositories.yaml` and rely on the `nexus-repositories-create` helper script included in the `cray/cray-nexus-setup` container
 image to facilitate creation.
 
+(`ncn-mw#`) If you are creating a new repository and want to add it to Zypper on the current node you can run (replacing `REPO_NAME` as appropriate):
+
+```bash
+zypper ar --no-gpgcheck https://packages.local/repositorys/REPO_NAME REPO_NAME
```

Comment:
```suggestion
zypper ar https://packages.local/repositorys/REPO_NAME REPO_NAME
```
Remove the `--nog-gpgcheck`, instead add a blockquote telling the user to add `--no-gpgcheck` if the repo is unsigned.